### PR TITLE
Improve title and subtitle

### PR DESCRIPTION
**Pull request by rgaiacs**
**This required #26.**

**This is just CSS changes.**

swc

### CODE REVIEW

File: css/swc.css [hunk 1]
```diff
@@ -53,8 +53,20 @@ h1, h2, h3, h4, h5, h6  {
...
-    margin:40px 0px;
-    border-bottom:1px solid #515151;
+    display: inline;
+    margin: 40px 0px;
+}
+
+/* Subtitle use the same font of title (set up by bootstrap.css) */
+h2.subtitle {
+    display: inline;
+    font-size: 38.5px;
```
Comment (gvwilson) [inline css/swc.css:63]: That's a very specific font size... :-)

File: css/swc.css [hunk 2]
```diff
@@ -53,8 +53,20 @@ h1, h2, h3, h4, h5, h6  {
...
+    margin: 40px 0px;
+}
+
+h2.subtitle::before {
+    content: "-";
```
Comment (gvwilson) [inline css/swc.css:68]: Why the '-' before?

Comment (rgaiacs) [inline css/swc.css:68]: > Why the '-' before?

I want something to separate the title and the subtitle and thought that '-' was
suitable. Do you prefer ':' or another markup?

Comment (gvwilson) [inline css/swc.css:68]: I thought it was enough that they were on different lines and different 
sizes.

Comment (rgaiacs) [inline css/swc.css:68]: I'm OK with different lines and different sizes.

**Comment by gvwilson** [comment]
What's the status of this one?

**Comment by rgaiacs** [comment]
@gvwilson Sorry for the delay. I thought that you are -1 to this:

> I thought it was enough that they were on different lines and different sizes.

Could you close this if this is the case or send me a description of how do you like the title and the subtitle.

**Pull request closed without merging**